Ghost-Ashu.md

```diff
@@ -0,0 +1,34 @@
+---
+name: "@Ghost-Ashu"
+project: "RealTime GPS Data Logger with GMaps"
+---
+---
+#### Email: ashutoshkumarsingh951@gmail.com
+---
+# RealTime GPS Data Logger with Goggle Maps
+
+## Summary
+
+Have You ever wonder! How Gobal Positioning System Tracks and Logs our Cordinates. Here  is my Project Walkthrough, where i will be building a realtime GPS Data Logger by using  Google Maps API and NodeMCU ESP8266.
+
+## Plan
+
+The Latitude and Longitude on the Display of 3.5inch TFT SPI LCD(ILI9488). It'll act as interface with GPS Module and NodeMCU ESP8266 also it will display the GPS data (Latitude, longitude, date and time). I have Tried to Collect from all the Sources in Building my Project
+ESP8266 Showing the Device position on the Display Screen. Using the Google Cloud platform i will be using the Static API of G- Maps. Display the geographic location of a user or device on a Google map.
+
+## Circuit And Architecture
+![20221224_124036 (2)](https://user-images.githubusercontent.com/41056892/209425577-d997ff2b-fc51-44ff-b319-5751af7988ad.png)
+
+
+## Budget
+
+| Product                                          | Supplier/Link                                                                   | Cost   |
+| -------------------------------------------------| --------------------------------------------------------------------------------| ------ |
+| Arduino Uno Rev3                                 |     https://store.arduino.cc/products/arduino-uno-rev3                          | $25.60 |
+| NodeMCU ESP8266 (AMICA-CP2102 driver)            |     https://robocraze.com/products/nodemcu-esp8266-amica-cp2102-driver          | $4.10  |
+|BN-282 GPS Module(NMEA-0183)                      |     https://bit.ly/3WxojpM                                                      | $53.71 |
+|3.5 inch TFT SPI LCD Display Resolution 480 x 320 |     https://amzn.eu/d/3HccX7w						                                       | $64.55 |
+|Jumper Wires					                             |     https://a.co/d/aHmKe1U							                                         | $6.98  |
+|SOLDERING IRON - 60W                              |     https://www.digikey.in/short/5mdr5vdh                                       | $16.47 |
```

Comment:
If you don't already own other soldering supplies such as a stand for the soldering iron, solder sucker, solder wick, tip cleaner, flux, stand, etc., you may want to consider a kit instead of just a soldering iron.

Reply:
Arduino Uno Rev3 can be used in my project, as it is a microcontroller board based on the ATmega328 microcontroller. One way it could be used to collect GPS data from a GPS module, process the data to extract the desired information (e.g. latitude, longitude, altitude), and then send the data to a remote server or cloud platform over a wired or wireless connection. To use the Arduino Uno in this project, you would need to connect it to the GPS module using a serial communication interface such as UART or I2C.
i already have my casing for my project. and Necessary other requirements. This is my Dream Project as this would my first GPS module Project that i have worked in years. i would be happy if u agree to approve my Project.